framework/include/image_data.hpp

```diff
@@ -7,87 +7,164 @@ namespace gvk
 	class image_data_implementor;
 
 
-	// interface of image_data type, used for abstraction and implementor in bridge pattern
+	/** Interface of image_data type, used for abstraction and implementor in bridge pattern
+	* This class should only be derived by the image_data and image_data_implementor classes
+	*/
 	class image_data_interface
 	{
 	public:
-		// type that represents the size of 1D, 2D, and 3D images
-		typedef vk::Extent3D extent_type;
+		/** type that represents the size of 1D, 2D, and 3D images
+		*/
+		using extent_type = vk::Extent3D;
```

Comment:
This is the default formatting in VS 2019. Is there an option to change it? Otherwise I would recommend leaving the formatting as is, as it would create a lot of extra work and a large source of inconsistency in the long run.

Reply:
Hmm, I would hope that it can be configured differently in VS, but I don't know how to from the top of my head. I guess, in my case Resharper formats it like I have requested.
This issue is not suuuuuper important. I'm sure that it can be fixed in a cleanup step at a later point in time, too.

Reply:
I looked at the config options and also searched for a solution but couldn't find any.